scripts/update-rules.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const recommendedRulesFile = require('../lib/config/recommended');
```

Comment:
nit: You could do `const { rules: recommendedRules } =` here.

Reply:
Good idea, fixed.